jaffle_shop/models/final/finance/fnl_returned_order_value.sql

```diff
@@ -0,0 +1,13 @@
+WITH returned_orders AS (
+  SELECT 
+    orders.customer_id AS customer_id
+    orders.amount as total_amount
```

Comment:
I don't think this SQL would run - do you know why?

Reply:
Ah, missing comma, thanks

Reply:
I think it won't run because you're referencing an alias (orders) without assigning it first. Also, FYI we use leading rather than trailing commas:

`SELECT 
  customer_id
  , SUM(COALESCE(total_amount, 0)) AS total_value
FROM returned_orders
GROUP BY customer_id`